### DISCUSSION TITLE
Individual Events

### DISCUSSION THREAD

**Discussion by wdwithmp** [totaldebug/atomic-calendar-revive#400] [Feature Requests]
Hi Mate, first let me thank you for the awesome work in maintaining this add-on. It's a life saver. I am new to HA and also learning as I go, so this maybe a noob question to you but I felt I needed to ask it anyway 

So the calendar component in HA does not support multiple events, specially if it overlaps. There is also no easy way to get all the events in a day, for example if you want to have a "morning brief" where you use TTS to read out the events in your calendar while you have your coffee (I maybe wrong) so given this add-on basically has all that information is there a way to get this information so that it can be used in a manner that we can use TTS to read out the days events

I did read all of the documentation, yours the initial developers but I couldn't get an idea if it is possible 

Again there maybe a way out there, that I don't know of, if there is I apologise for the post

**Comment by marksie1988** [comment]
Hey @wdwithmp,

The Lovelace Cards are only capable of taking the information from HomeAssistant and displaying this on the interface, as far as i'm aware they would not be able to manipulate TTS.

In order to manipulate TTS I assume this would require an integration / add-on rather than a lovelace card, however this is not something that I have used before so am not 100% sure.

**Reply by wdwithmp** [reply]
Hi Mate, sorry rookie mistake, I was not clear 
Is there a way to use this add-on to read all the events in a day? 
The add-on shows all the events and it starts and shows progress of each event (including overlapping events) - so is there a way to (or entity) to get all the events for a given day?

if there is a way to read the events in a day, then we can use that entity to trigger stuff or even use the event description to forward to TTS

**Reply by marksie1988** [reply]
So an entity for a calendar just pulls x amount of events depending on the settings in .google_calendar.yml you would only be able to read out from TTS the events in that entity, i'm not sure about putting filters on the entity, i'm sure you can but that is not something that I have done before. 

I think that questions like this are less related to the card and more the functionality of HomeAssistant, for that I would recommend jumping over the [DrZzs Discord](https://discord.me/drzzs) as there are people over there with way more experience on what Home Assistant can do
